assets/marketplace/Makefile

```diff
@@ -14,7 +14,7 @@ MARKETPLACE_AMI_NAME ?=
 AWS_REGION ?= us-west-2
 
 # Teleport version
-TELEPORT_VERSION ?= 4.1.6
+TELEPORT_VERSION ?= 4.2.1
```

Comment:
In the release instructions:

https://gravitational.slab.com/posts/practices-12mko276#release-checklist-teleport

We have the following:

``
Update VERSION in the Makefile, run make version.go & commit both file changes to the target branch
```

Should `assets/marketplace/Makefile` derive its version from there, or be included in the list of places to update?

Reply:
I suppose it should probably be updated manually, although it isn’t super important as it’s just a default and we override the `VERSION` variable in the Jenkins job that ultimately runs the AMI builds.

I don’t really want to start introducing dependencies which reach across Makefiles.

Reply:
> I don’t really want to start introducing dependencies which reach across Makefiles.

That makes a lot of sense.  Since this is a default that is overriden, and you seem to be the only one significantly involved in this process, I'll leave it up to you if you choose to add assets/marketplace/Makefile to the instructions.

I'm keeping an eye out for all these random changes that seem 4.2 related, since I was in charge of the 4.2 release. Trying to consolidate much of the followup work because there is a surprising amount of it, and our bus factor isn't high.